packages/tests-unit/tests/converters/aws-apigw-v1.test.ts

```diff
@@ -0,0 +1,264 @@
+        body: Readable.toWeb(Readable.from(Buffer.from(""))),
+        headers: {
+          single: "test",
+          multi: ["test1", "test2"],
+        },
+        isBase64Encoded: false,
+        statusCode: 200,
+      })) as APIGatewayProxyResult;
+
+      expect(response.headers).toStrictEqual({
+        single: "test",
+      });
+      expect(response.multiValueHeaders).toStrictEqual({
+        multi: ["test1", "test2"],
+      });
+    });
+  });
+});
+
+describe("convertFrom", () => {
+  it("Should convert a simple APIGatewayProxyEvent", async () => {
+    const event: APIGatewayProxyEvent = {
+      body: JSON.stringify({ message: "Hello, world!" }),
+      headers: {
+        "content-type": "application/json",
+      },
+      multiValueHeaders: {},
+      httpMethod: "POST",
+      isBase64Encoded: false,
+      path: "/",
+      pathParameters: null,
+      queryStringParameters: null,
+      multiValueQueryStringParameters: null,
+      stageVariables: null,
+      requestContext: {
+        identity: {
+          sourceIp: "::1",
+        },
+      } as any,
+      resource: "",
+    };
+
+    const response = await converter.convertFrom(event);
+
+    expect(response).toEqual({
+      type: "core",
+      method: "POST",
+      rawPath: "/",
+      url: "/",
+      body: Buffer.from('{"message":"Hello, world!"}'),
+      headers: {
+        "content-type": "application/json",
+      },
+      remoteAddress: "::1",
+      query: {},
+      cookies: {},
+    });
+  });
+
+  it("Should handle multiValueHeaders", async () => {
+    const event: APIGatewayProxyEvent = {
+      body: JSON.stringify({ message: "Hello, world!" }),
+      headers: {},
+      multiValueHeaders: {
+        test: ["test1", "test2"],
+      },
+      httpMethod: "POST",
+      isBase64Encoded: false,
+      path: "/",
+      pathParameters: null,
+      queryStringParameters: null,
+      multiValueQueryStringParameters: null,
+      stageVariables: null,
+      requestContext: {
+        identity: {
+          sourceIp: "::1",
+        },
+      } as any,
+      resource: "",
+    };
+
+    const response = await converter.convertFrom(event);
+
+    expect(response).toEqual({
+      type: "core",
+      method: "POST",
+      rawPath: "/",
+      url: "/",
+      body: Buffer.from('{"message":"Hello, world!"}'),
+      headers: {
+        test: "test1,test2",
+      },
+      remoteAddress: "::1",
+      query: {},
+      cookies: {},
+    });
+  });
+
+  it("Should handle queryStringParameters and multiValueQueryStringParameters", async () => {
+    const event: APIGatewayProxyEvent = {
+      body: JSON.stringify({ message: "Hello, world!" }),
+      headers: {},
+      multiValueHeaders: {},
+      httpMethod: "POST",
+      isBase64Encoded: false,
+      path: "/",
+      pathParameters: null,
+      queryStringParameters: {
+        test: "test",
+      },
+      multiValueQueryStringParameters: {
+        test: ["test"],
```

Comment:
What happens when the same `key (test)` has different values? Do we want to merge the two? Is that even possible?
eg:
```
      queryStringParameters: {
        test: "test",
      },
      multiValueQueryStringParameters: {
        test: ["test-multi"],
      }
```

Reply:
I guess it shouldn't be possible, hopefully aws has some kind of validation on this to avoid this kind of case.